{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/vue/components/images/LazyImage.css

```diff
@@ -0,0 +1,69 @@
+/*
+|--------------------------------------------------------------------------
+| Image
+|--------------------------------------------------------------------------
+| @namespace: img-
+|
+*/
+.img-Image {
+  position: relative;
+
+  margin: 0;
+
+  background-color: #0e0e0e;
+  overflow: hidden;
+}
+
+.img-Image_Media {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  width: 100%;
+  height: 100%;
+}
+
+.img-Image_Image {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  width: 100%;
+  height: 100%;
+
+  opacity: 0;
+
+  transition: opacity 0.5s linear;
+
+  @supports (object-fit: cover) {
+    object-fit: cover;
+  }
```

Comment:
Is any of the object-fit stuff (and corresponding hacks to make it work in IE11) needed? By definition, it will always cover the full size of the container as it is absolutely positioned within it.